Source/Streamstone/Stream.Operations.cs

```diff
@@ -31,7 +31,7 @@ public Stream Execute()
 
                 try
                 {
-                    table.Execute(insert.Prepare());
+                    table.ExecuteAsync(insert.Prepare()).Wait();
```

Comment:
Sure, my comment was simply to point out that as the code stands now, I don't think it's a good idea to pull it in. I'm all for removing sync APIs :+1: